dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -305,14 +311,31 @@ def sync_course
           member[:roles].include?(Policies::Lti::CONTEXT_LEARNER_ROLE)
         end
 
+        total_students += student_count
+        total_sections += 1
+
         result[:all][section_id] = {
           name: section[:name],
           size: student_count,
         }
       end
     end
 
+    metadata = {
```

Comment:
[This PR](https://github.com/code-dot-org/code-dot-org/pull/57563) fixes the way we count students during syncs. Right now, this will just count everything in the NRPS response, instead of the specific sections being synced (sections where the user isn't a teacher won't be synced). Your call if you want to add this now and have some numbers be slightly off, or hold off until after my PR. I could even just add it to mine.

Reply:
Sounds good, I'll rebase off your PR once it gets merged and refactor this code. Looks like I'll just be able to get the section_count and student_count out of the returned sync result hash

Reply:
Just merged it in, so you should be good to rebase off staging

Reply:
Pulled in the changes and refactored this code.